bots/make-checkout

```diff
@@ -35,6 +35,7 @@ def main():
     parser = argparse.ArgumentParser(description="Fetch and checkout specific revision")
     parser.add_argument("-v", "--verbose", action="store_true", help="Verbose output")
     parser.add_argument("--base", nargs='?', help="Base branch that revision is for")
+    parser.add_argument("--stable", action="store_true", help="Test stable branch")
```

Comment:
It's not intuitively clear what that does, or why it even needs to be an option. What is a "stable" branch in the generic context of our bots CI? The positional "ref" argument should be all that's needed. I. e. that can be "master" or "rhel-8.1" or a SHA  if it's a PR.